aspnetcore/signalr/messagepackhubprotocol.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Use MessagePack Hub Protocol in SignalR for ASP.NET Core
+author: 
+description: Learn how to add MessagePack Hub Protocol to ASP.NET Core SignalR.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: 
+ms.custom: mvc
+ms.date: 06/04/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/messagepack
+---
+
+# Use MessagePack Hub Protocol in SignalR for ASP.NET Core
+
+This article assumes the reader is familiar with the topics covered in [Get Started](xref:signalr/get-started).
+
+## What is MessagePack
+
+MessagePack is a binary serialization format that is fast and compact. It is useful when performance and bandwidth are a concern because it creates smaller packets compared to JSON. Because it is a binary format, it will be unreadable when looking at network traces and logs unless you pass the bytes through a MessagePack parser. You can see more information about the format at https://msgpack.org/index.html. SignalR has added built-in support for the MessagePack format and added APIs for the client and server to use.
+
+## Configure MessagePack on the Server
```

Comment:
Maybe add a note about adding support for the protocol globally vs per hub

Reply:
You have to add support globally